infra/common/src/main/java/org/apache/shardingsphere/infra/datasource/state/exception/UnavailableDataSourceException.java

```diff
@@ -35,4 +35,8 @@ public final class UnavailableDataSourceException extends ShardingSphereServerEx
     public UnavailableDataSourceException(final SQLException cause) {
         super(ERROR_CATEGORY, ERROR_CODE, "Data source unavailable.", cause);
     }
+    
+    public UnavailableDataSourceException(final SQLException cause, final String databaseName) {
+        super(ERROR_CATEGORY, ERROR_CODE, String.format("Data source unavailable from `%s` .", databaseName), cause);
```

Comment:
Hi @sunkai-cai ,
Is there an extra space after `%s`?

Reply:
oh, i remove it.